dds_web/forms.py

```diff
@@ -62,6 +66,20 @@ class RegistrationForm(flask_wtf.FlaskForm):
     confirm = wtforms.PasswordField("Repeat password")
     submit = wtforms.SubmitField("submit")
 
+    def validate_username(self, username):
+        user = models.User.query.filter_by(username=username.data).first()
+        if user:
+            raise wtforms.validators.ValidationError(
+                "That username is taken. Please choose a different one."
+            )
+
+    def validate_email(self, email):
+        email = models.Email.query.filter_by(email=email.data).first()
+        if email:
+            raise wtforms.validators.ValidationError(
+                "That email is taken. Please choose a different one."
```

Comment:
Can you even choose the email at signup?

Reply:
No, it's not suppose to, but it's good to have this check here anyway.